### PR TITLE
Add toc, more details, call out dynamic referencing as an advanced to…

### PR DESCRIPTION
**Pull request by thebioengineer**
…pic.

### CODE REVIEW

**Review by mariev — COMMENTED** [review]
Wondering what audience we expect to target with this vignette? I like having minimal examples to help conceptualize the an intro to these new concepts, but the emphasis on validate-while-developing means that the concepts are not easily abstracted to the other modes. It may not be immediately obvious to some the other modes make {valtools} immediately useful for validation outside of development 😬 

Looking ahead, will there be a separate vignette for each mode? if so, what can we do now to be repetitive about shared concepts such as requirements, etc. without having to re-implement text at a low level?

File: vignettes/starting-validated-package.Rmd [hunk 1]
```diff
@@ -20,90 +26,154 @@ knitr::opts_chunk$set(
...
+In general, the process consists of the following steps, and the
+vignette as specific examples for each.
+
+-   Create A New R Package
+-   Add Requirements
+-   Package Development
+-   Change log
+-   Testing
+-   Authoring Validation Reports
+-   Validation
```
[the remark this answers is not in the document]
Comment (thebioengineer) [reply vignettes/starting-validated-package.Rmd:43]: Totally reasonable - when writing it I don't see the TOC - so didnt realize that. We could remove the bulleted list in the vignette.

File: vignettes/starting-validated-package.Rmd [hunk 2]
```diff
@@ -20,90 +26,154 @@ knitr::opts_chunk$set(
...
 
-This vignette steps through the process of creating an R package that is intended to be 
-validated, by extending `usethis` and `devtools` functionality to provide a 
-seamless validation process. 
+There is a combination of explaining how to use {valtools} and tutorial
+steps for the reader to follow along in creating a simple R package that
+could undergo validation.
```
[the remark this answers is not in the document]
Comment (thebioengineer) [reply vignettes/starting-validated-package.Rmd:47]: a bit - I wanted to have people see the concepts applied in valtools and how they could apply it to their packages/other projects.

File: vignettes/starting-validated-package.Rmd [hunk 3]
```diff
@@ -235,71 +351,121 @@ test_that("1.2",{
...
+documentation by creating a final report that can run and gather all the
+information that was generated in the prior steps to create the final
+report.
 
-`vt_use_report()` is used to create the validation report rmarkdown. This is 
-populated in the working directory specified in `validation.yml`, which for
-packages defaults to the `vignettes`, and creates a template based on the 
-report Rmd name specified in `validation.yml`, which defaults to `validation.Rmd`.
+The report is written in Rmarkdown so that it is flexible and dynamic.
+{valtools} provides a function to create a template that can be worked
```
[the remark this answers is not in the document]
Comment (thebioengineer) [reply vignettes/starting-validated-package.Rmd:366]: True! didn't make that connection. Maybe its my mental model where knitr/rmarkdown knits the file into the output doc, what would that original file be called?

File: vignettes/starting-validated-package.Rmd [hunk 4]
```diff
@@ -285,33 +281,33 @@ When editing the report, some key functions to know for extending the report inc
   
 Keep in mind, the report is an Rmarkdown, so there is no limit to editing and customization, and templates.
 
-## Running a validation report
+## Running a Validation Report
```
[the remark this answers is not in the document]
Comment (thebioengineer) [reply vignettes/starting-validated-package.Rmd:443]: We can go into more detail with them in different vignettes for sure. I wanted to mainly call out we support a few different methods at the moment.

[the remark this answers is not in the document]
Comment (thebioengineer) [reply vignettes/starting-validated-package.Rmd:432]: opened issue #137 to make sure we don't lose this

**Pull request merged**